sagemaker-featurestore/feature_store_client_side_encryption.ipynb

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
<p>method name should be cast_object_to_string?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Fixed.